### PR TITLE
chore(ci): add GitHub Issue templates

### PR DESCRIPTION
**Pull request by shinGangan**
### 🔗 Linked issue

related nuxt-modules/sitemap#323

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

I added the same issue templates as I added in Nuxt Scripts.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/02-bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    attributes:
+      label: 🐛 The bug
+      description: What isn't working? Describe what the bug is.
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: 🛠️ To reproduce
+      description: 'A reproduction of the bug. Please use the Nuxt OG Image starter https://stackblitz.com/edit/nuxt-starter-pkwfkx?file=pages%2Findex.vue'
+      placeholder: https://stackblitz.com/[...]
```
Comment (shinGangan) [inline .github/ISSUE_TEMPLATE/02-bug-report.yml:18]: #### Note

For the StackBlitz environment, I am using the same one as Nuxt Scripts.
(I don't think it's a high priority to create a dedicated environment, so I think it's fine to leave it as it is for now and handle it in a separate PR.)

**Comment by shinGangan** [comment]
Hi @harlan-zw , please review when you have a time 🙏

**Comment by harlan-zw** [comment]
Thank you! I updated the stackblitz demo links.

**Comment by shinGangan** [comment]
> Thank you! I updated the stackblitz demo links.

@harlan-zw 

Thanks for updating the demo links!

**Pull request merged**